src/crypto/README.md

```diff
@@ -31,7 +31,7 @@ throughout the rest of the code.
 The rest of the files are structured by their function, as detailed in the
 following table:
 
-| File (*.h/*.cc)      | Description |
+| File (\*.h/\*.cc)    | Description |
```

Comment:
This one fixes a straight-up bug. This currently renders as italics without the `*` because that's how markdown would process it.